pb/protos/profile.proto

```diff
@@ -56,4 +49,20 @@ message Profile {
         string large    = 4;
         string original = 5;
     }
+
+    message Stats {
+        uint32 followerCount  = 1;
+        uint32 followingCount = 2;
+        uint32 listingCount   = 3;
+        uint32 avgRating      = 4;
+        uint32 numRatings     = 5;
```

Comment:
I'd also make `averageRating` last. Keep all the `xxxCount` fields together towards the top of the struct.

Also, why isn't the rating a `float` type?